<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Caroline-PC" name="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist" tests="161" failures="19" timestamp="3 Feb 2018 18:17:40 GMT" time="4359.786" errors="0">
  <testcase name="F1_onlyWhite1_admin5" time="23.752" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite6_admin7" time="7.805" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E1_onlyWhite_me7" time="28.699" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite7_admin5" time="23.583" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite2_admin5" time="24.175" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite2_admin2" time="28.785" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 2001拨打991000,预期呼入失败,实际呼入成功">
      <![CDATA[java.lang.AssertionError:  2001拨打991000,预期呼入失败,实际呼入成功
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.F1_onlyWhite2_admin2(Blacklist_Whitelist.java:1939)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- F1_onlyWhite2_admin2 -->
  <testcase name="F1_onlyWhite9_admin6" time="18.157" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite6_admin7" time="7.737" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite3_admin1" time="61.242" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E1_onlyWhite_me3" time="26.467" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite6_admin3" time="43.237" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite6_admin5" time="23.732" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E2_onlyWhite_me1" time="26.480" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="J6_blackMe_deleteAll_yes" time="13.400" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite6_admin1" time="86.804" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E3_onlyWhite_me2" time="23.598" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite8_admin3" time="29.198" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 2001拨打991000,预期呼入失败,实际呼入成功">
      <![CDATA[java.lang.AssertionError:  2001拨打991000,预期呼入失败,实际呼入成功
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.F1_onlyWhite8_admin3(Blacklist_Whitelist.java:2536)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- F1_onlyWhite8_admin3 -->
  <testcase name="F1_onlyWhite4_admin6" time="23.570" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite2_admin2" time="30.051" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite1_admin4_cancel" time="28.608" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite9_admin2" time="28.832" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 2001拨打991000,预期呼入失败,实际呼入成功">
      <![CDATA[java.lang.AssertionError:  2001拨打991000,预期呼入失败,实际呼入成功
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.F1_onlyWhite9_admin2(Blacklist_Whitelist.java:2632)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- F1_onlyWhite9_admin2 -->
  <testcase name="F1_onlyWhite6_admin2" time="30.481" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="A1_addExtensions" time="51.344" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite1_admin6" time="18.144" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="I5_whiteMe_deleteAll_no" time="9.825" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite3_admin3" time="42.920" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite2_admin1" time="63.293" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite2_admin3" time="42.933" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="J4_blackMe_deletePart_yes" time="13.367" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite2_admin3" time="29.733" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite2_admin6" time="23.557" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite3_admin6" time="18.195" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E2_onlyWhite_me3" time="23.631" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="I1_whiteMe_deleteOne_no" time="52.784" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite4_admin4_cancel" time="27.825" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite4_admin3" time="23.730" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite3_admin7" time="7.757" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite7_admin1" time="27.044" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="D1_black_me1" time="37.427" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite5_admin1" time="58.981" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite8_admin4_cancel" time="14.556" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite1_admin5" time="29.277" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 2001拨打991000,预期呼入失败,实际呼入成功">
      <![CDATA[java.lang.AssertionError:  2001拨打991000,预期呼入失败,实际呼入成功
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.F2_onlyWhite1_admin5(Blacklist_Whitelist.java:2807)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- F2_onlyWhite1_admin5 -->
  <testcase name="E1_onlyWhite_me1" time="28.591" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite3_admin5" time="18.166" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E1_onlyWhite_me8" time="24.059" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite1_admin1" time="58.549" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite5_admin5" time="18.193" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite2_admin6" time="23.541" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite4_admin2" time="18.867" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="H5_whiteAdmin_deleteAll_no" time="9.805" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite4_admin5" time="23.584" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="H3_whiteAdmin_deletePart_no" time="21.663" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite5_admin3" time="18.583" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite3_admin2" time="18.156" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite4_admin7" time="15.667" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite4_admin4_cancel" time="14.885" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite7_admin4_cancel" time="27.720" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite1_admin3" time="18.191" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="G6_blackAdmin_deleteAll_yes" time="13.377" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite4_admin6" time="18.813" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="I4_whiteMe_deletePart_yes" time="13.375" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite2_admin4_cancel" time="28.609" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite2_admin7" time="28.153" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite3_admin5" time="24.174" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite4_admin3" time="35.891" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite8_admin2" time="23.610" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite7_admin6" time="23.756" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite8_admin1" time="27.033" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite1_admin6" time="23.501" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite7_admin7" time="7.528" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E2_onlyWhite_me4" time="26.447" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite5_admin7" time="7.720" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="G3_blackAdmin_deletePart_no" time="18.621" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E3_onlyWhite_me1" time="26.445" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E2_onlyWhite_me7" time="26.701" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite5_admin3" time="18.159" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite9_admin1" time="56.866" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite7_admin4_cancel" time="14.531" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E2_onlyWhite_me9" time="28.844" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 2006拨打991000,预期呼入失败,实际呼入成功">
      <![CDATA[java.lang.AssertionError:  2006拨打991000,预期呼入失败,实际呼入成功
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.E2_onlyWhite_me9(Blacklist_Whitelist.java:1706)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- E2_onlyWhite_me9 -->
  <testcase name="J2_blackMe_deleteOne_yes" time="26.416" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite6_admin6" time="23.509" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="G4_blackAdmin_deletePart_yes" time="13.357" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite4_admin5" time="23.727" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E2_onlyWhite_me8" time="23.650" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite3_admin6" time="23.528" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite1_admin4_cancel" time="27.390" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E3_onlyWhite_me4" time="26.488" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite4_admin2" time="24.035" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite2_admin5" time="23.745" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite7_admin3" time="24.088" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="H6_whiteAdmin_deleteAll_yes" time="13.395" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite7_admin6" time="18.811" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="H1_whiteAdmin_deleteOne_no" time="63.087" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite7_admin1" time="81.654" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="H4_whiteAdmin_deletePart_yes" time="13.372" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite3_admin1" time="37.827" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite5_admin2" time="18.240" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite9_admin5" time="29.307" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 2001拨打991000,预期呼入失败,实际呼入成功">
      <![CDATA[java.lang.AssertionError:  2001拨打991000,预期呼入失败,实际呼入成功
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.F1_onlyWhite9_admin5(Blacklist_Whitelist.java:2685)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- F1_onlyWhite9_admin5 -->
  <testcase name="F2_onlyWhite6_admin2" time="28.777" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 2001拨打991000,预期呼入失败,实际呼入成功">
      <![CDATA[java.lang.AssertionError:  2001拨打991000,预期呼入失败,实际呼入成功
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.F2_onlyWhite6_admin2(Blacklist_Whitelist.java:3382)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- F2_onlyWhite6_admin2 -->
  <testcase name="F2_onlyWhite1_admin7" time="7.616" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite3_admin4_cancel" time="27.905" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="J1_blackMe_deleteOne_no" time="35.960" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite7_admin2" time="28.797" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 2001拨打991000,预期呼入失败,实际呼入成功">
      <![CDATA[java.lang.AssertionError:  2001拨打991000,预期呼入失败,实际呼入成功
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.F2_onlyWhite7_admin2(Blacklist_Whitelist.java:3498)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- F2_onlyWhite7_admin2 -->
  <testcase name="F1_onlyWhite1_admin2" time="28.831" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 2001拨打991000,预期呼入失败,实际呼入成功">
      <![CDATA[java.lang.AssertionError:  2001拨打991000,预期呼入失败,实际呼入成功
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.F1_onlyWhite1_admin2(Blacklist_Whitelist.java:1830)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- F1_onlyWhite1_admin2 -->
  <testcase name="F1_onlyWhite9_admin4_cancel" time="28.736" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite6_admin4_cancel" time="27.873" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite1_admin2" time="26.690" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 2001拨打991000,预期呼入失败,实际呼入成功">
      <![CDATA[java.lang.AssertionError:  2001拨打991000,预期呼入失败,实际呼入成功
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.F2_onlyWhite1_admin2(Blacklist_Whitelist.java:2754)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- F2_onlyWhite1_admin2 -->
  <testcase name="I3_whiteMe_deletePart_no" time="18.656" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite6_admin3" time="29.645" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E2_onlyWhite_me5" time="24.147" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite8_admin5" time="28.800" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 2001拨打991000,预期呼入失败,实际呼入成功">
      <![CDATA[java.lang.AssertionError:  2001拨打991000,预期呼入失败,实际呼入成功
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.F1_onlyWhite8_admin5(Blacklist_Whitelist.java:2565)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- F1_onlyWhite8_admin5 -->
  <testcase name="J3_blackMe_deletePart_no" time="21.685" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite1_admin3" time="41.867" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite9_admin3" time="18.188" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E1_onlyWhite_me9" time="28.802" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 2001拨打991000,预期呼入失败,实际呼入失败">
      <![CDATA[java.lang.AssertionError:  2001拨打991000,预期呼入失败,实际呼入失败
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.E1_onlyWhite_me9(Blacklist_Whitelist.java:1601)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- E1_onlyWhite_me9 -->
  <testcase name="F2_onlyWhite7_admin5" time="18.638" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E1_onlyWhite_me6" time="23.536" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite6_admin6" time="28.807" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 2001拨打991000,预期呼入失败,实际呼入成功">
      <![CDATA[java.lang.AssertionError:  2001拨打991000,预期呼入失败,实际呼入成功
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.F2_onlyWhite6_admin6(Blacklist_Whitelist.java:3435)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- F2_onlyWhite6_admin6 -->
  <testcase name="G2_blackAdmin_deleteOne_yes" time="25.025" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite5_admin6" time="23.723" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite5_admin4_cancel" time="27.928" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="I6_whiteMe_deleteAll_yes" time="13.338" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite5_admin7" time="7.592" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="G5_blackAdmin_deleteAll_no" time="9.883" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite2_admin1" time="77.137" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E1_onlyWhite_me4" time="23.800" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite3_admin3" time="18.613" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite1_admin1" time="25.821" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite6_admin1" time="38.745" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E3_onlyWhite_me3" time="29.459" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 2006拨打991000,预期呼入失败,实际呼入成功">
      <![CDATA[java.lang.AssertionError:  2006拨打991000,预期呼入失败,实际呼入成功
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.E3_onlyWhite_me3(Blacklist_Whitelist.java:1744)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- E3_onlyWhite_me3 -->
  <testcase name="F1_onlyWhite6_admin5" time="24.209" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite8_admin7" time="22.555" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite3_admin4_cancel" time="28.487" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite4_admin7" time="7.747" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="H2_whiteAdmin_deleteOne_yes" time="25.017" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E2_onlyWhite_me2" time="29.303" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 3001拨打3000,预期呼入失败,实际呼入成功">
      <![CDATA[java.lang.AssertionError:  3001拨打3000,预期呼入失败,实际呼入成功
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.E2_onlyWhite_me2(Blacklist_Whitelist.java:1631)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- E2_onlyWhite_me2 -->
  <testcase name="F1_onlyWhite6_admin4_cancel" time="28.850" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite5_admin2" time="29.126" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 2001拨打991000,预期呼入失败,实际呼入成功">
      <![CDATA[java.lang.AssertionError:  2001拨打991000,预期呼入失败,实际呼入成功
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.F2_onlyWhite5_admin2(Blacklist_Whitelist.java:3239)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- F2_onlyWhite5_admin2 -->
  <testcase name="F1_onlyWhite8_admin6" time="23.566" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite5_admin4_cancel" time="27.972" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite1_admin7" time="27.572" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite7_admin3" time="18.132" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E3_onlyWhite_me6" time="23.577" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite3_admin2" time="29.222" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 2001拨打991000,预期呼入失败,实际呼入成功">
      <![CDATA[java.lang.AssertionError:  2001拨打991000,预期呼入失败,实际呼入成功
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.F1_onlyWhite3_admin2(Blacklist_Whitelist.java:2046)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- F1_onlyWhite3_admin2 -->
  <testcase name="F1_onlyWhite9_admin7" time="20.645" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite4_admin1" time="26.381" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite5_admin5" time="18.600" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="G1_blackAdmin_deleteOne_no" time="81.012" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E1_onlyWhite_me5" time="28.836" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 2001拨打991000,预期呼入失败,实际呼入失败">
      <![CDATA[java.lang.AssertionError:  2001拨打991000,预期呼入失败,实际呼入失败
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.E1_onlyWhite_me5(Blacklist_Whitelist.java:1553)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- E1_onlyWhite_me5 -->
  <testcase name="E2_onlyWhite_me6" time="23.519" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite2_admin7" time="7.763" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite2_admin4_cancel" time="27.494" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite5_admin6" time="18.170" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="J5_blackMe_deleteAll_no" time="9.888" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite5_admin1" time="77.101" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F1_onlyWhite7_admin2" time="18.201" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="I2_whiteMe_deleteOne_yes" time="25.045" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite3_admin7" time="7.416" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="E3_onlyWhite_me5" time="28.840" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist">
    <failure type="java.lang.AssertionError" message=" 3001拨打3000,预期呼入失败,实际呼入成功">
      <![CDATA[java.lang.AssertionError:  3001拨打3000,预期呼入失败,实际呼入成功
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist.E3_onlyWhite_me5(Blacklist_Whitelist.java:1774)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- E3_onlyWhite_me5 -->
  <testcase name="E1_onlyWhite_me2" time="23.834" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
  <testcase name="F2_onlyWhite4_admin1" time="85.970" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist"/>
</testsuite> <!-- com.yeastar.swebtest.testcase.DetailCase.pbxcase.Blacklist_Whitelist -->
